src/Talismans.ts

```diff
@@ -405,11 +405,16 @@ export const updateTalismanAppearance = (i: number) => {
     }
 }
 
+// Attempt to buy a fixed number of levels (number varies based on
+// ascension). Returns true if all levels were bought -- false otherwise
```

Comment:
This should probably return true if any level was bought - it was supposed to represent any change to talisman shards, so that the autobuyer can update rune levels and inventory (but only once instead of for every talisman)

Reply:
Ah, rereading the autobuyer code that makes more sense -- for some reason i thought the autobuyer was running in a loop and only breaking out when this returns true (it is in a loop, but over the talis types :) and there's no breaking involved). Fixed